emotesMain/src/main/java/io/github/kosmx/emotes/main/config/ClientConfig.java

```diff
@@ -64,7 +64,8 @@ public ClientConfig(){
     //public final EmoteHolder[] fastMenuEmotes = new EmoteHolder[8];
 
     public BiMap<UUID, InputKey> emoteKeyMap = new BiMap<>();
-    public UUID[] fastMenuEmotes = new UUID[8];
+    public ConfigEntry<Integer> fastMenuPage = new IntegerConfigEntry("fastMenuPage", 0, false, expert);
```

Comment:
I don't think you should use a config entry to store the last used page.
I think a static variable in the ModernChooseWheel is better.